configure.py

```diff
@@ -1627,23 +1627,35 @@ def make_bin_override():
       ' '.join([pipes.quote(arg) for arg in original_argv]) + '\n')
 os.chmod('config.status', 0o775)
 
+
```

Comment:
Stray whitespace change?

Reply:
No. I meant to better delineate the lines that generate `config.mk`.
Optimally I'd wrap this code into a function, but that will just conflict with https://github.com/nodejs/node/pull/26725